system/autoware_error_monitor/config/diagnostic_aggregator/_discard.yaml

```diff
@@ -1,9 +1,15 @@
-analyzers:
-  speaker:
-    type: diagnostic_aggregator/DiscardAnalyzer
-    path: speaker
-    contains: ": speaker"
-  joystick:
-    type: diagnostic_aggregator/DiscardAnalyzer
-    path: joystick
-    contains: ": Joystick"
+/**:
+  ros__parameters:
+    path: system
+    discard:
```

Comment:
I needed to change the layout here in order for the `DiscardAnalyzers` to be picked up else these would be entirely skipped for some reason by the Diagnostics Aggregator.